src/main.js

```diff
@@ -1,14 +1,29 @@
 // Globals
-window.worldData
 window.game = new Phaser.Game(1000, 600, Phaser.Auto, '');
 
-startGame();
+var Main = function() {};
+var worldData;
```

Comment:
I meant put it in `game.worldData` or something similar but I'm not sure what the best way to do this is in javascript. It's okay the way it is now, but when we implement the worldData object we should figure out how to place it.